webapp/components/DeleteData/DeleteData.vue

```diff
@@ -55,22 +55,19 @@
           <ds-space margin-bottom="xx-small" />
           <ds-flex :gutter="{ base: 'xx-small', md: 'small', lg: 'large' }">
             <ds-flex-item :width="{ base: '100%', sm: '100%', md: '100%', lg: 1.75 }">
-              <ds-input
-                v-model="enableDeletionValue"
-                @input="enableDeletion"
-                class="enable-deletion-input"
-              />
+              <ds-input v-model="enableDeletionValue" class="enable-deletion-input" />
```

Comment:
nice change, I think I added this, but having a `v-model` means there is no need for an `@input` since their values are bound